src/models/NFTAttributes.ts

```diff
@@ -32,26 +34,63 @@ export const defaultNeverminedNFTType = {
 
 export class NFTAttributes {
 
-    ercType: ERCType // The type of ERC used
-
-    nftType: NeverminedNFTType // The Nevermined implementetion of the NFT used
-
+    /**
+     * The type of ERC used (721 or 1155)
+     * @see {@link https://ethereum.org/en/developers/docs/standards/tokens/erc-721/}
+     * @see {@link https://ethereum.org/en/developers/docs/standards/tokens/erc-1155/}
+     */
+    ercType: ERCType
+
+    /**
+     * The Nevermined implementetion of the NFT used.
+     * A part of what type of ERC is based, Nevermined provides different NFT implementations to fit in different scenarios.
+     * This attribute allow to specify between the different Nevermined NFT types
+     */
+    nftType: NeverminedNFTType
+
+    /**
+     * The address of the deployed NFT Contract
+     */
     nftContractAddress: string
 
-    cap?: BigNumber // max number of nfts that can be minted, 0 means uncapped
-
-    preMint?: boolean // if the asset is pre-minted
-
-    nftMetadataUrl?: string // URL to the metadata definition of the NFT contract
-
-    nftTransfer?: boolean // The asset is transferred (true) or minted (false) with Nevermined contracts
-
+    /**
+     * Max number of nfts that can be minted, 0 means uncapped
+     */
+    cap?: BigNumber
+
+    /**
+     * If the asset is pre-minted
+     */
+    preMint?: boolean 
+
+    /**
+     * URL to the metadata definition of the NFT contract
+     */
+    nftMetadataUrl?: string
+
+    /**
+     * The asset is transferred (true) or minted (false) with Nevermined contracts
+     */
+    nftTransfer?: boolean
+
+    /**
+     * If true means the NFT works as a subscription
+     */
     isSubscription?: boolean
```

Comment:
I don't think this is needed since we should get this information from the nft types

Reply:
The intention of this flag is to identify a subscription even when the nft type is not `nft721Subscription`.
For example a POAP that is a subscription too. In that case it would be:
```
nftType: nft721POAP
isSubscription: true
```